File: examples/java/hello/function/src/main/java/ru/nikolaymatrosov/Handler.java
package ru.nikolaymatrosov;

import yandex.cloud.sdk.functions.Context;
import yandex.cloud.sdk.functions.YcFunction;

import java.io.PrintStream;
import java.io.UnsupportedEncodingException;

public class Handler implements YcFunction<Integer, String> {
    @Override
    public String handle(Integer i, Context c) {
        PrintStream out = null;
        try {
            out = new PrintStream(System.out, true, "UTF-8");
        } catch (UnsupportedEncodingException e) {
            throw new RuntimeException(e);
        }
        out.println("Hello, World! Привет, Мир!");
        return String.valueOf(i);
    }
}

File: examples/csharp/apigw/ApiGW/ApiGWHandler/Handler.cs
using System.Text.Json;
using Yandex.Cloud.Functions;

namespace ApiGWHandler;

public class Handler : YcFunction<ApiGatewayRequest, Response>
{
    public Response FunctionHandler(ApiGatewayRequest request, Context c)
    {
        Console.WriteLine($"Function name: {c.FunctionId}");
        Console.WriteLine($"Function version: {c.FunctionVersion}");
        Console.WriteLine($"Service account token: {c.TokenJson}");
        Console.WriteLine($"Request body: {request.Body}");
        var data = JsonSerializer.Deserialize<Request>(request.Body, new JsonSerializerOptions()
            {
                PropertyNamingPolicy = JsonNamingPolicy.CamelCase
            }
        );
    
        string name = data?.Name ?? "World";

        string body = JsonSerializer.Serialize(
            new Dictionary<string, string>
            {
                { "message", $"Hello, {name}!" },
            });

        Console.WriteLine($"Response body: {body}");

        return new Response(200, body, new Dictionary<string, string>
        {
            { "Content-Type", "application/json" },
        });
    }
}